src/vm/reflectioninvocation.cpp

```diff
@@ -1323,13 +1340,23 @@ FCIMPL5(Object*, RuntimeMethodHandle::InvokeMethod,
         gc.retVal = Nullable::NormalizeBox(gc.retVal);
     }
     else
-    if (retType == ELEMENT_TYPE_VALUETYPE)
+    if (hasValueTypeReturn || hasRefReturnAndNeedsBoxing)
```

Comment:
-> `if (hasValueTypeReturn)`

Reply:
No, the ` || hasRefReturnAndNeedsBoxing` has to stay since the if clause now contains the logic for the `hasRefReturnAndNeedsBoxing` handling as well. (I did this way to avoid creating another duplicate `Nullable::NormalizeBox` call along with its associated TODO).

Reply:
ok